### PR TITLE
Bug/2030 set throttling to zero

### PR DESCRIPTION
**Pull request by kzangeli**
Fixes issue #2030.
Actually, that issue was already fixed. This PR just adds a functest for it.

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -30,6 +30,7 @@
 - Fix: Better error returned on attribute not found in GET /v2/entities/<entity-id>/attrs/<attr-name>/value (Issue #2220)
 - Fix: Distinguish between strings and numbers in q string filter (Issue #1129)
 - Fix: proper response for 'GET /v2/entities/<id>?attrs=<attrName>' and 'GET /v2/entities/<id>/attrs?attrs=<attrName>' when <attrName> doesn't exist (#2241)
+- Fix: throttling set to zero (Issue #2030)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: This seems to be another case of "it was fixed in the past but we don't know when"... We should use the .test to locate the newest failing Orion version in this order: 1.2.1, 1.2.0, 1.1.0, 1.0.0, etc.

Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: Yes, this is another one of those :-(
Will perform those tests and hopefully I can tell when the bug was fixed ...

Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: Fixed in dbfd21e

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -31,6 +31,7 @@
 - Fix: Better error returned on attribute not found in GET /v2/entities/<entity-id>/attrs/<attr-name>/value (Issue #2220)
 - Fix: Distinguish between strings and numbers in q string filter (Issue #1129)
 - Fix: proper response for 'GET /v2/entities/<id>?attrs=<attrName>' and 'GET /v2/entities/<id>/attrs?attrs=<attrName>' when <attrName> doesn't exist (#2241)
+- Fix: throttling set to zero (Issue #2030) (This issue was actually fixed for the release 1.2.0)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Given this is a change in an already release version, this entry should be moved to the 1.2.0 section in contextBroker.spec.

Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: Fixed in ec7947d

**Comment by fgalan** [comment]
LGTM

**Pull request merged**